kustomize/commands/internal/util/util.go

```diff
@@ -42,7 +42,7 @@ func GlobPatternsWithLoader(fSys filesys.FileSystem, ldr ifc.Loader, patterns []
 		if len(files) == 0 {
 			loader, err := ldr.New(pattern)
 			if err != nil {
-				log.Printf("%s has no match", pattern)
+				return nil, fmt.Errorf("%s has no match", pattern)
```

Comment:
suggestion: Maybe worth including the original error?

Reply:
That is a good suggestion. Thanks! 
I add it.